.gitattribute

```diff
@@ -1 +0,0 @@
-mock/ext-api-dyson/* linguist-vendored
```

Comment:
this one seems not working, atlas is still a major js project

Reply:
Hmm, file got deleted unintentionally when I merged master branch, strange.